src/views/BTC2XFLI/BTC2XFLI.tsx

```diff
@@ -30,13 +33,13 @@ const Btc2xFliProductPage = (props: { title: string }) => {
     token: Bitcoin2xFlexibleLeverageIndex,
     components: components,
     balance: btcfliBalance,
-    supplyCap: supplyCap,
+    supplyCap: btcfliSupplyCap,
     currentSupply: btcfliTotalSupply,
   }
   const { account } = useWallet()
 
   const isApproachingSupplyCap = btcfliTotalSupply
-    ?.div(supplyCap)
+    ?.div(btcfliSupplyCap as BigNumber)
```

Comment:
is there a scenario here where we're dividing by 0? and would that break anything?

Reply:
Good point, I think only place possible would be getSupplyCap on error, I just proposed changing return on catch to 1 instead of 0

Reply:
IMO it'd be better to address that here, since this now has a dependency on getSupplyCap returning 1. That's just a nit pick though.